Header  header
float32 pos_x
float32 pos_y
float32 pos_z
float32 roll
float32 pitch
float32 yaw
bool    active